### ISSUE TITLE
[Feature Request] Add ability to remove custom search engines en mass.

### ISSUE THREAD

**Issue by chris37879** [gregsadetsky/chrome-dont-add-custom-search-engines#24]
Would adding a feature that allows users to bulk remove custom search engines that build up in the settings be out of place? I ask because I just installed the extension and needed to also delete old ones and this could be a nice feature for on-boarding new users.

There's a console script someone posted on superuser that can remove it, not sure if extensions can operate on chrome:// urls.

https://superuser.com/questions/1141135/google-chrome-remove-all-other-search-engines

**Comment by cxw42** [comment]
Chris, thanks for the link and feedback!  I think this would be a nice feature to include.  Unfortunately, extensions are not allowed to run on chrome: pages.  (If they were, for example, a malicious extension could present itself from being uninstalled.)  Extensions can't even link to chrome: pages, in fact, although they can use the extension api to open chrome: pages.

Since this extension originates from SE, there are no license issues with making this code available in the extension for people to copy and paste.  It would be another point of required updating, though, so including just the link would have reduced maintenance burden.  @gregsadetsky what do you think?

**Comment by gregsadetsky** [comment]
A prompt after the extension's installation to remove all search engines is a great idea -- thank you for the suggestion! But yeah, we're limited (for good reason) in what we can do around chrome URLs.

I imagine that we could add a "thank you for installing" screen and either link to stackoverflow or show the instructions right there (for "Advanced Users"). I'm not the biggest fan of post-installation prompts/web pages popping up, though.

I'm not sure. What about adding a item to the extension icon's menu titled "Remove all search engines", and instead of doing it, showing the instructions to do so..?

**Comment by chris37879** [comment]
I agree with the concerns about a post install popup that is just
informational isn't great. Maybe just put the instructions in the view you
can open by clicking the button. That's where I looked first for a thing to
remove them after installing the extension.

**Comment by cxw42** [comment]
I agree with a screen that opens when the user clicks the extension's button, and that has a link to the instructions.  Currently clicking the button does nothing, so we have that space available.
